src/libraries/System.Runtime.Numerics/src/System/Numerics/Complex.cs

```diff
@@ -13,7 +14,11 @@ namespace System.Numerics
     /// </summary>
     [Serializable]
     [TypeForwardedFrom("System.Numerics, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089")]
-    public readonly struct Complex : IEquatable<Complex>, IFormattable
+    public readonly struct Complex
+        : IEquatable<Complex>,
+          IFormattable,
```

Comment:
Are we leaving IFormattable here to avoid impacting lookup perf by changing the ordering?

Reply:
Yep. Trying to minimize impact of these new interfaces overall.

There's probably some tweaking we may want to do long term, including for the primitives.